taca/analysis/analysis_nanopore.py

```diff
@@ -1,4 +1,5 @@
 """Nanopore analysis methods for TACA."""
+from genericpath import isdir
```

Comment:
don't think you use this in the end? `os.path.isdir` is a different thing right?

Reply:
Yeah, you're right. Can't recall when this was added 😅 I'll remove it 👍